tasks/zadanie_22.php

```diff
@@ -31,3 +31,14 @@
 */
 
 $clients = $params[0]; // tej linijki nie ruszamy :)
+
+function welcomeMessage ($client) : string{
```

Comment:
tutaj brakuje zdefiniowanego typu ;) 
```suggestion
function welcomeMessage (array $client) : string{
```